### PR TITLE
chore(tests): add kubernetes provider tests

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Add tests for Kubernetes provider.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/providers/kubernetes/lib/service/service_test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            audit_config=None,
+            audit_resources=[],
+            audit_metadata=Audit_Metadata(
+                services_scanned=0,
+                expected_checks=[],
+                completed_checks=0,
+                audit_progress=0,
+            ),
+        )
+        return audit_info
```
Comment (jfagoagas) [inline tests/providers/kubernetes/lib/service/service_test.py]: I think you can use the one from audit_info utils here.

File: tests/providers/kubernetes/kubernetes_provider_test.py [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    @patch("kubernetes.config.load_kube_config")
+    @patch("kubernetes.config.load_incluster_config")
+    @patch("kubernetes.config.list_kube_config_contexts")
+    def test_setup_session(
+        self,
+        mock_list_kube_config_contexts,
+        mock_load_incluster_config,
+        mock_load_kube_config,
+        mock_api_client,
+        mock_rbac_api,
```
Comment (jfagoagas) [inline tests/providers/kubernetes/kubernetes_provider_test.py:21]: What is the purpose of this?

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: tests/providers/kubernetes/lib/service/service_test.py (hunk 1 above)
Comment (MrCloudSec) [reply tests/providers/kubernetes/lib/service/service_test.py]: Done

**Review by jfagoagas — APPROVED** [review]
Good one, thanks @sergargar 🚀

**Pull request merged**